### DISCUSSION TITLE
[Design] - Background color for Terminal Persistent Sessions

### DISCUSSION THREAD

**Discussion by shivapoudel** [primer/github-vscode-theme#295] [Q&A]
Any workaround to control this background color for terminal session restored??
![image](https://user-images.githubusercontent.com/3774827/153702609-5df23691-a6a9-4cd6-a8f3-a0e620e218b9.png)

**Comment by simurai** [comment]
Hmm.. not sure if there is way to override the background color. What should the alternative be?

There seems to be a way to [disable the feature](https://github.com/microsoft/vscode/issues/136566#issuecomment-962504728) tough.
